.github/workflows/main.yaml

```diff
@@ -38,29 +40,49 @@ jobs:
       with:
         fetch-depth: 2
 
-    # Skip CI for commits with a message containing the skip-ci string.
```

Comment:
Is ok to remove ... I need a better skip code.

I very minor comment regarding:

```
   runs-on: [ windows-2019, windows-2016 ]
```

The current x64 vs x32 bit build design was shaped by buildmaster limitation.

In an ideal world we would  have a matrix in which we define which windows version to run and also decide which bit version... so that we could also build x32 on 2019. 

so the 32 vs 64 build decision is not made by brink.sh  / pythia.sh but by the env variables.

Reply:
Agreed. Saved as an issue: https://github.com/chevah/pythia/issues/15.